src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1258,41 +1235,63 @@ bistTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tree::pt
 }
 
 /*
- * TEST #11
- */
-void
-vcuKernelTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tree::ptree& _ptTest)
+* Helper function to perform an initial check of whether a test is supported on device
+*/
+bool
+check_test_supported(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tree::ptree& _ptTest)
 {
-  runTestCase(_dev, "xcl_vcu_test.exe", _ptTest.get<std::string>("xclbin"), _ptTest);
-}
+  // If there is no extra test file, skip to testHandle.
+  auto file = _ptTest.get<std::string>("file");
+  if (file.compare("") == 0)
```

Comment:
Isn't this the same as:
```suggestion
  if (file.empty())
```